src/config/gobjects.rs

```diff
@@ -199,6 +203,18 @@ fn parse_object(toml_object: &Value, concurrency: library::Concurrency) -> GObje
     };
     let members = Members::parse(toml_object.lookup("member"), &name);
     let properties = Properties::parse(toml_object.lookup("property"), &name);
+    let derives = {
+        let mut v = Vec::new();
+        if let Some(configs) = toml_object.lookup("derive").and_then(|val| val.as_array()) {
```

Comment:
Maybe this better be as `Derives::parse(`, same as previous?

Reply:
Yes indeed